mlir/CMakeLists.txt

```
@@ -13,9 +13,11 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ standard to conform to")
 
 find_package(MLIR REQUIRED CONFIG)
+find_package(MHLO REQUIRED)
```

As CMake searches a config cmake script here (`MHLOConfig.cmake`), 
```suggestion
find_package(MHLO REQUIRED CONFIG)
```